presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -116,6 +116,47 @@ Memory Management Properties
     This is the amount of memory set aside as headroom/buffer in the JVM heap
     for allocations that are not tracked by Presto.
 
+
+Query Management Properties
+---------------------------
+
+``query.max-execution-time``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``duration``
+    * **Default value:** ``100 days``
+
+    The maxiumum allowed time for a query to be actively executing across the
+    cluster on all nodes, before it is terminated.
+
```

Comment:
- can be overridden with `query_max_execution_time` 
- "on all nodes" may be misleading, suggesting this is "concurrent time" (wall time * number of nodes); This limit is "normal time" (just wall time since actual execution start). Queue, analysis, planning time are excluded (of which queuing time should be mentioned here)